x-pack/test/functional/apps/security/management.js

```diff
@@ -61,13 +61,21 @@ export default function ({ getService, getPageObjects }) {
         it('Clicking save in create user section brings user back to listing', async () => {
           await PageObjects.security.clickCreateNewUser();
 
-          await testSubjects.setValue('userFormUserNameInput', 'new-user');
-          await testSubjects.setValue('passwordInput', '123456');
-          await testSubjects.setValue('passwordConfirmationInput', '123456');
-          await testSubjects.setValue('userFormFullNameInput', 'Full User Name');
-          await testSubjects.setValue('userFormEmailInput', 'example@example.com');
+          // await testSubjects.setValue('userFormUserNameInput', 'new-user');
+          // await testSubjects.setValue('passwordInput', '123456');
+          // await testSubjects.setValue('passwordConfirmationInput', '123456');
+          // await testSubjects.setValue('userFormFullNameInput', 'Full User Name');
+          // await testSubjects.setValue('userFormEmailInput', 'example@example.com');
```

Comment:
I would remove all the comments here and below the call to `createUsers`.

We can use git if we ever want to revert.